### DISCUSSION TITLE
求 战争之人突击小队2

### DISCUSSION THREAD

**Discussion by 972933728** [Gamuxorg/bbs#391] [投递和请求游戏]
如题

**Comment by wangling12** [comment]
无linux版本，暂不考虑打包，自行参考
https://appdb.winehq.org/objectManager.php?sClass=application&iId=16796

**Comment by 972933728** [comment]
了解 谢谢打包Shank




------------------ 原始邮件 ------------------
发件人: "wangling12"<notifications@github.com>;
发送时间: 2019年6月26日(星期三) 晚上9:10
收件人: "Gamuxorg/bbs"<bbs@noreply.github.com>;
抄送: "fox"<aaa972933728@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [Gamuxorg/bbs] 求 战争之人突击小队2 (#55)




无linux版本，暂不考虑打包，自行参考
 https://appdb.winehq.org/objectManager.php?sClass=application&iId=16796
